packages/react-components/src/badge/src/Badge.jsx

```diff
@@ -51,7 +51,7 @@ export function InnerBadge(props) {
             )}
             ref={forwardedRef}
         >
-            {content}
+            <span>{content}</span>
```

Comment:
Yeah, I am still again this.

I think when possible we should avoir extra element.

I have a hard time buying it's required to correctly center the text since in the previous version of Orbit UI we we're able to do it without the extra SPAN and this is same use case except the root element is a DIV instead of a BUTTON.

https://sg-storybook.netlify.app/?path=/docs/components-button--default-story

Reply:
It works there since we don't have a badge smaller that 16px. It fails at 14px since the content is 15px.

Reply:
Looking at Atlassian they have the same issue but avoid it by not having a badge smaller than 16px.

Reply:
Do we need a badge having 14px? What is the use case?

Reply:
When in a Tag but I guess 16px wouldn't be bad either